documentation.html

```diff
@@ -65,7 +65,7 @@
         <a href="" class="sidebar-items">Badge</a>
       </li>
       <li class="sidebar-list-items">
-        <a href="" class="sidebar-items">Button</a>
+        <a href="/button/button.html" class="sidebar-items">Button</a>
       </li>
       <li class="sidebar-list-items">
         <a href="" class="sidebar-items">Badge</a>
```

Comment:
class names could be named better.
`sidebar-list-item` and `sidebar-link` could be used instead.